README.md

```diff
@@ -20,34 +20,82 @@ Policy-based control for Kubernetes deployments.
     ```bash
     kubectl create namespace opa
     ```
+2. Add permissions in your Cluster for OPA/kube-mgmt
+   
+    ```bash
+    cat > permissions-kube-mgmt.yaml <<'EOF'
+    # Grant OPA/kube-mgmt read-only access to resources. This let's kube-mgmt
+    # replicate resources into OPA so they can be used in policies.
+    kind: ClusterRoleBinding
+    apiVersion: rbac.authorization.k8s.io/v1
+    metadata:
+      name: opa-viewer
+    roleRef:
+      kind: ClusterRole
+      name: view
+      apiGroup: rbac.authorization.k8s.io
+    subjects:
+    - kind: Group
+      name: system:serviceaccounts:opa
+      apiGroup: rbac.authorization.k8s.io
+    ---
+    # Define role for OPA/kube-mgmt to update configmaps with policy status.
+    kind: Role
```

Comment:
@tsandall just double checked this and the ClusterRoleBinding (view) is also required for listing the configmap(s) in kube-mgmt. Otherwise I'm facing the following:

```
E1219 20:04:37.956541       1 reflector.go:201] github.com/open-policy-agent/kube-mgmt/pkg/policies/configmap.go:124: Failed to list *v1.ConfigMap: unknown (get configmaps)
```